CoreGraphUtilities/src/au/gov/asd/tac/constellation/graph/utilities/widgets/IconChooser.java

```diff
@@ -337,6 +341,7 @@ private void saveButtonActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FI
                 }
             } catch (final IOException ex) {
                 NotifyDisplayer.display(String.format("Error writing icon file %s:%n%s", selectedFile.toString(), ex.getMessage()), NotifyDescriptor.ERROR_MESSAGE);
+                LOGGER.log(Level.WARNING, ex.getLocalizedMessage());
```

Comment:
I think I would make the level `SEVERE` since the Notify displayer is displaying an error message